### PR TITLE
Adding mark-nonconverted-reads

### PR DESCRIPTION
**Pull request by mattsoup**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Thanks for your help!

### CODE REVIEW

**Comment by mattsoup** [comment]
@bioconda/core I think this is ready to go.  It would also be great if I could be added as a member here. Thanks!

File: recipes/mark-nonconverted-reads/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+requirements:
+  host:
+    - python >=3
+
+  run:
+    - python >=3
+    - pysam
+
+build:
+  noarch: python
```
Comment (dpryan79) [inline recipes/mark-nonconverted-reads/meta.yaml]: Can you add a build number (0)? That makes upgrades easier.

**Comment by dpryan79** [comment]
I've added you to the team, go ahead and merge after adding a build number! Thanks

File: recipes/mark-nonconverted-reads/meta.yaml (hunk 1 above)
Comment (mattsoup) [reply recipes/mark-nonconverted-reads/meta.yaml]: Done!

**Pull request merged**